main/templatetags/noindex_meta.py

```diff
@@ -0,0 +1,16 @@
+"""Blocks search engine indexing for non-production environments."""
+from django import template
+from django.utils.safestring import mark_safe
+from django.conf import settings
+
+register = template.Library()
+
+
+@register.simple_tag()
+def noindex_meta():
+    """Adds in noindex for non-production environments."""
+    return (
+        mark_safe("""<meta name="robots" content="noindex">""")
+        if settings.ENVIRONMENT not in ("production", "prod")
+        else None
```

Comment:
It works fine when we return the meta tag but has some issues when we return `None`. It adds `None` as a string and moves it with all the meta tags and the title below it to the `<body>`. It works fine if we return an empty string instead of None.
<img width="409" alt="Screen Shot 2023-03-08 at 2 33 35 PM" src="https://user-images.githubusercontent.com/52656433/223676803-6e6cbfec-8643-4cfd-b0f3-713d29691fa6.png">

Reply:
Nice catch!